Assets/Scripts/Foliage/SpawnFoliage.cs

```diff
@@ -11,7 +11,7 @@
 
 public class SpawnFoliage : MonoBehaviour
 {
-    const int prefabLimit = 20000;
```

Comment:
Only thing that's bugging me. Why is this changed? Andy why in this PR?

Reply:
Fixed, must have been something I forgot when testing